mindsdb/integrations/handlers/financial_modeling_prep_handler/financial_modeling_handler.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+from typing import Dict
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+)
+from mindsdb.integrations.handlers.financial_modeling_prep_handler.financial_modeling_tables import HistoricalPriceTable
+
+from mindsdb.utilities import log
+import requests
+_FINANCIAL_MODELING_URL = 'https://financialmodelingprep.com/api/v3/'
```

Comment:
This variable does not seem to be in use anywhere. Let's either use it in the code (within the `connect()` method) or remove it completely. I prefer the first option because that will allow us to change it easily in the future if required. Let's also give it a more meaningful name, maybe `FMP_BASE_URL`?